src/styles/GlobalStyle.js

```diff
@@ -248,7 +248,7 @@ const GlobalStyle = createGlobalStyle`
     }
 
     &[target='_blank'] {
-      cursor: ne-resize;
+      cursor: pointer;
```

Comment:
I did this on purpose! Could you remove?

Reply:
Yes! Is this a design choice? or is there a specific reason? (Asking for my own knowledge)